test/python/circuit/test_isometry.py

```diff
@@ -69,7 +69,7 @@ def test_isometry(self, iso):
         unitary = result.get_unitary(qc)
         iso_from_circuit = unitary[::, 0 : 2**num_q_input]
         iso_desired = iso
-        self.assertTrue(matrix_equal(iso_from_circuit, iso_desired, ignore_phase=True))
+        self.assertTrue(matrix_equal(iso_from_circuit, iso_desired, ignore_phase=False))
```

Comment:
self.assertTrue(np.allclose(iso_from_circuit, iso_desired))

Reply:
Thank you, @adjs . You are right. I will replace `matrix_equal` with `np.allclose`.